### PR TITLE
Configuring PAM for K8S prod

### PR DESCRIPTION
**Pull request by tamas-molnar**
- moved read env config to global-configs -see https://github.com/Financial-Times/upp-global-configs/pull/21
- set base url for validation
  - for production envs: manually in the app-config
  - for team envs: read from deployment param (depends on Jenkins job feature)
- added PAM secrets for production, see lastpass
- updated secrets in K8S CT

### CODE REVIEW

File: helm/publish-availability-monitor/templates/deployment.yaml [hunk 1]
```diff
@@ -65,20 +65,21 @@ spec:
             secretKeyRef:
               name: publish-availability-monitor-secrets
               key: notifications_push.api_key
+        {{- $base_url := default .Values.envs.validation_endpoints.base_url .Values.cluster.delivery.url }}
```
Comment (sbuliarca) [inline helm/publish-availability-monitor/templates/deployment.yaml]: I was wrong on this one.. the parameters should be inverted.

File: helm/publish-availability-monitor/app-configs/publish-availability-monitor_publishing_prod.yaml [hunk 2]
```diff
@@ -20,7 +20,5 @@ envs:
     methode_article_internal_components_mapper: "__methode-article-internal-components-mapper/map"
     video_mapper: "__upp-next-video-mapper/map"
     wordpress_mapper: "__wordpress-article-mapper/map"
-    base_url: "https://upp-k8s-delivery-test-eu.ft.com/"
+    base_url: "http://upp-prod-delivery.ft.com/"
```
Comment (sbuliarca) [inline helm/publish-availability-monitor/app-configs/publish-availability-monitor_publishing_prod.yaml]: This needs to be https:

File: helm/publish-availability-monitor/app-configs/publish-availability-monitor_publishing.yaml [hunk 3]
```diff
@@ -9,7 +9,10 @@ envs:
   internal_components_url: "/__document-store-api/internalcomponents/"
   lists_url: "/__document-store-api/lists/"
   notifications_url: "/__notifications-rw/content/notifications?type=all"
-  notifications_push_url: "/content/notifications-push?monitor=true"
+  # I had to split the URL into 2 parts because I couldn't find a way to put the exact string \& in the value
+  # This way I moved handling that value in the template chart where it seems it doesn't have a problem with it.
+  notifications_push_url:  "/content/notifications-push?monitor=true"
+  notifications_pursh_url_param2: "type=all"
```
Comment (tosan88) [inline helm/publish-availability-monitor/app-configs/publish-availability-monitor_publishing.yaml]: There's a typo in "push"

Comment (sbuliarca) [reply helm/publish-availability-monitor/app-configs/publish-availability-monitor_publishing.yaml]: good catch!

**Pull request merged**